### PR TITLE
fix: easier amount input

### PR DESCRIPTION
**Pull request by mpetrunic**
## Related Issue Or Context

Closes: #150 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in sygma-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Review by Lykhoyda — APPROVED** [review]
Looks good, please fix the tests

File: packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts [hunk 1]
```diff
@@ -103,6 +103,9 @@ export class ResourceAmountSelector extends BaseComponent {
   };
 
   _validateAmount(amount: string): boolean {
+    if (amount === '') {
```
Comment (wainola) [inline packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts:106]: if I try to pass a number like `13.45` the `.` will be treated as empty string, and the amount will be reseted to zero. Firefox treats this differently. The dot repeats the actual value that is currently int the input, and then continues. This is why this block works on firefox but not on chromium based browsers

**Review by wainola — CHANGES_REQUESTED** [review]

File: packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts (hunk 1 above)
Comment (mpetrunic) [reply packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts:106]: I'm sorry, I don't follow. I just tried 13.45 on both chrome and firefox and it works perfectly

Comment (wainola) [reply packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts:106]: my bad, I wasn't aware that the browser was taking into consideration the localization for the type of symbol you use for decimal places, in my case `,` and not `.`, so everything is fine here. First time using `,` for decimal places though 😄

**Comment by wainola** [comment]
there is a small issue with the cursor when deleting the amount, it goes to the start of the input and leave some numbers there

**Comment by Lykhoyda** [comment]
@mpetrunic @wainola I think we can keep the type number for now, it's not causing big issues. If we want to change it, we can open a new issue. I will approve this one

**Review by Lykhoyda — APPROVED** [review]

**Pull request merged**